CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## Version 1.3.0
+
+## Features
+- Password history setting allows to save user's last used password so that it may not be used again. Maximum number of history saved is 20, which can be configured [#4866](https://github.com/appwrite/appwrite/pull/4866)
```

Comment:
No need to explain minimum, I think. Tho if we do, let's also mention default value.

Reply:
default is 0 which mean it's not enabled, do we need to mention that?

Reply:
I think we can, just in case we expose this API in the future.

Reply:
updated